src/components/CodeBlock/normalize.ts

```diff
@@ -29,49 +29,55 @@ const stripComment = line =>
     ``,
   );
 
-const containsDirective = line =>
+const containsDirective = (line: string) =>
```

Comment:
I think the typing is a little too obv. we should have some level of implicit typing for simple params.

Reply:
TypeScript can't infer param types

Reply:
Since this is a function arg, the type is required.

Reply:
Learned this the hard way when he walked over and explained it to me :)